### ISSUE TITLE
Look Up/Down not working

### ISSUE THREAD

**Issue by Jaffacakelover** [jval1972/MD2-Viewer#1]
The keyboard shortcuts for Look Up / Look Down are not working.
The Quick info box says that these actions are bound to Page Up / Page Down.

Turned scroll lock on/off on my keyboard, but it no makes difference.

Tested on:
MD2 Model Viewer 1.3.1.1.0.1
Windows 10 (64-bit)

**Comment by jval1972** [comment]
The PageUp/Page Down buttons are working in the numeric keyboard. (9 for PgUp & 3 for PgDown). They work regardless the Num Lock on/off.
I can not test it right know in a keyboard with dedicated PgUp/PgDown keys since I only have access to a laptop. Can you try the PgUp/PgDown in the numeric keyboard and tell me if they work?

**Comment by Jaffacakelover** [comment]
Yes, that's it! Tested on desktop and Look Up and Look Down are bound to Keypad 9 and Keypad 3.
Just like you, my laptop doesn't have a numerical keypad: Could you add support for the "normal" page up / page down / plus / minus keys in future? Or some mouselook controls?
